<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_scr_dish_main_quit</class>
 <widget class="QDialog" name="w_scr_dish_main_quit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="lds_toolbar_button" name="pushButton_selectAll">
         <property name="text">
          <string>全选</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="lds_toolbar_button" name="pushButton_clearAll">
         <property name="text">
          <string>全不选</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="lds_toolbar_button" name="pushButton_up">
         <property name="text">
          <string>上一页</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="lds_toolbar_button" name="pushButton_down">
         <property name="text">
          <string>下一页</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="lds_tableView" name="tableView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="lineEdit_vch_dishname"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>名称</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>数量</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="lds_doublespinbox" name="doubleSpinBox_num_dish"/>
         </item>
         <item row="1" column="3">
          <widget class="lds_doublespinbox" name="doubleSpinBox_num_dish_org"/>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QLineEdit" name="lineEdit_vch_reason"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>备注</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="lds_key3x4" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="lds_toolbar_button" name="pushButton_quit_bill_kitchen_view">
       <property name="text">
        <string>确认退单</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_quit_bill">
       <property name="text">
        <string>整单退货</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_tableView</class>
   <extends>QTableView</extends>
   <header>lds_tableview.h</header>
  </customwidget>
  <customwidget>
   <class>lds_doublespinbox</class>
   <extends>QDoubleSpinBox</extends>
   <header>lds_doublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_key3x4</class>
   <extends>QWidget</extends>
   <header>lds_key3x4.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_vch_dishname</tabstop>
  <tabstop>doubleSpinBox_num_dish</tabstop>
  <tabstop>doubleSpinBox_num_dish_org</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
